lua/nvim-tree.lua

```diff
@@ -324,6 +324,7 @@ local DEFAULT_OPTS = { -- BEGIN_DEFAULT_OPTS
   view = {
     width = 30,
     height = 30,
+    hide_root_folder = false,
```

Comment:
There was an extra `hide_root_folder` in the above table. It doesn't appear to be used, so I deleted it.

Reply:
good catch, it wasn't in the right place !